src/transport/primus/server.js

```diff
@@ -38,7 +38,7 @@ PrimusTransport.prototype._connect = function(url,port){
 	var address = url+':'+port;
 	if(!this._connections[address]){
 		this._connections[address] = new this._Studio.promise(function(resolve,reject){
-            var client = new _self._server.Socket(url+":"+port,_self.clientOpt);
+            var client = new _self._server.Socket('ws://' + url+":"+port,_self.clientOpt);
```

Comment:
I think we should start simple... so, if you want ssl you will have to use it in all process, if you dont want ssl then you must deploy all without ssl.